gammapy/detect/cwt.py

```diff
@@ -73,25 +73,29 @@ class CWT(object):
     background model to which was added the flux of the extragalactic background,
     and the exposure for the considered energy range were used.
 
+    Read more in ???.
 
     TODO: instead of storing all the arrays as data members,
     we could have a ``.data`` member which is a dict of images?
 
     Parameters
     ----------
-    min_scale : float
-        first scale used
-    nscales : int
-        number of scales considered
-    scale_step : float
-        base scaling factor
+    n_scale : int
+        Number of scales considered.
+    min_scale : double
+        First scale used.
+    scale_step : double
+        Base scaling factor.
     max_iter : int, optional (default 10)
         The maximum number of iterations os the CWT algorithm.
     tol : double, optional (default 1e-5)
         Tolerance for stopping criterion.
-
-    TODO: give references
-    Read more in ???.
+    nsigma (significance max???) : double, optional (default 3.0)
```

Comment:
@cdeil @adonath 
I think `nsigma` and `nsigmap` should be renamed. I understand what for I need this parameters, but I don't understand how to renamed them. Look at `CWT._compute_support()` method for better understanding  of how to use it, I left comments there.